ui/src/app/views/dataset/component.tsx

```diff
@@ -196,43 +219,73 @@ export const DataSet = withCurrentWorkspace()(class extends React.Component<
                 <ClrIcon shape='plus-circle' class='is-solid' style={styles.addIcon}
                   onClick={() => navigate(['workspaces', namespace, id,  'cohorts', 'build'])}/>
               </div>
-              {/*TODO: load cohorts and display here*/}
-              <div style={{height: '8rem'}}/>
+              <div style={{height: '10rem', overflowY: 'auto'}}>
+                {this.state.cohortList.length > 0 && !loadingResources &&
+                  this.state.cohortList.map(cohort =>
```

Comment:
nit: Don't need to check length explicitly, if the array is empty then nothing will get rendered.